lib/tasks/import.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require "csv"
+
+namespace :import do
+  desc "Import merritt arks from csv for UCSB ETDs"
+
+  task merritt_arks: :environment do
+    # To run the task on the ETD fixture at
+    # spec/fixtures/proquest/zipped/ETD_ucsb_0035D_13132.zip
+    # CSV file contents would be as below
+    # Proquest ID,Meritt Ark,UCSB Ark
+    # ProQuestID:0035D,ark:/13030/m00999g9,ark:/48907/f30865g5
+
+    puts "Beginning Import #{Time.zone.now}"
+    csv_text = File.read("tmp/ucsb_merritt_etds.csv")
```

Comment:
We probably shouldn't hardcode the file path; it can be an argument passed to the task.